ccloud/docs/ccloud-demos-overview.rst

```diff
@@ -0,0 +1,189 @@
+.. _ccloud-demos-overview:
+
+Confluent Cloud Demos
```

Comment:
suggest: `|ccloud|`

Reply:
The characters `=====` that indicate a heading can be used with substitution variables. Just make the underlying heading characters equal the length of the heading with the subst variable and sphynx works it out when processing the rst.